### PR TITLE
mock-array: clean up config a bit

### PR DESCRIPTION
**Pull request by oharboe**
No change, just better communication of intent and forwards compatible with changes to detailed routing args.

### CODE REVIEW

**Review by maliberty — APPROVED** [review]

File: flow/designs/asap7/mock-array/Element/config.mk [hunk 1]
```diff
@@ -29,7 +29,7 @@ export PDN_TCL                = $(FLOW_HOME)/platforms/asap7/openRoad/pdn/BLOCK_
 # If this design isn't quickly done in detailed routing, something is wrong.
 # At time of adding this option, only 3 iterations were needed for 0
 # violations.
-export DETAILED_ROUTE_ARGS=-bottom_routing_layer M2 -top_routing_layer M5 -save_guide_updates -verbose 1 -droute_end_iter 10
+export DETAILED_ROUTE_END_ITERATION = 10
```
Comment (maliberty) [inline flow/designs/asap7/mock-array/Element/config.mk]: "only 3 iterations" != 10

File: flow/designs/asap7/mock-array/config.mk [hunk 2]
```diff
@@ -59,7 +59,7 @@ power:
 # If this design isn't quickly done in detailed routing, something is wrong.
 # At time of adding this option, only 12 iterations were needed for 0
```
Comment (maliberty) [inline flow/designs/asap7/mock-array/config.mk]: "only 12 iterations" != 15

**Review by maliberty — COMMENTED** [review]
Are the comments useful?

**Comment by oharboe** [comment]
> Are the comments useful?

Tidied the up a bit and add two words about the motivation for limiting iterations

File: flow/designs/asap7/mock-array/Element/config.mk [hunk 3]
```diff
@@ -26,13 +26,9 @@ export IO_CONSTRAINTS         = designs/asap7/mock-array/Element/io.tcl
 
 export PDN_TCL                = $(FLOW_HOME)/platforms/asap7/openRoad/pdn/BLOCK_grid_strategy.tcl
 
-# If this design isn't quickly done in detailed routing, something is wrong.
-# At time of adding this option, only 3 iterations were needed for 0
-# violations.
-export DETAILED_ROUTE_ARGS=-bottom_routing_layer M2 -top_routing_layer M5 -save_guide_updates -verbose 1 -droute_end_iter 10
+# Detailed routing should be easy, limit iterations
+export DETAILED_ROUTE_END_ITERATION = 6
```
Comment (maliberty) [inline flow/designs/asap7/mock-array/Element/config.mk:30]: If it is easy then it will finish naturally without needing a limit.

**Review by maliberty — APPROVED** [review]

File: flow/designs/asap7/mock-array/Element/config.mk (hunk 3 above)
Comment (oharboe) [reply flow/designs/asap7/mock-array/Element/config.mk:30]: True, and this limit is to ensure that it stays that way...

**Pull request merged**